src/scancodes/set123/scancode.c

```
@@ -255,11 +247,14 @@ static void process_normal_code(uint8_t code, const scancode_config_t *config) {
     // Set-specific special codes
     switch (config->set) {
         case SCANCODE_SET1:
-            // Set 1: Break = make | 0x80
-            if (code < 0x80) {
-                handle_keyboard_report(code, true);
+            // Set 1: Break = make | 0x80, valid break range 0x81-0xD3
+            if (code == 0xAA || code == 0xFC) {
```

Clang (14.0.6)

[warning] repeated branch in conditional chain

(bugprone-branch-clone)